react-stopwatch-component/my-app/src/StopWatch.tsx

```diff
@@ -0,0 +1,51 @@
+import { FaPlay, FaPause } from 'react-icons/fa6';
+import { useState } from 'react';
+
+export function StopWatch() {
+  const [time, setTime] = useState(0);
+  const [intervalId, setIntervalId] = useState<NodeJS.Timeout>();
+
+  const running = intervalId ? true : false;
```

Comment:
Nice that you calculate `running` from `intervalId`.

The tricky way to do this is `const running = !!intervalId;`